adapters/coap-vertx-base/src/main/java/org/eclipse/hono/adapter/coap/DefaultDeviceResolver.java

```diff
@@ -55,19 +55,10 @@
 
 
 /**
- * A Hono Credentials service based implementation of Scandium's authentication related interfaces.
+ * A Hono Credentials service based implementation of Scandium's PSK authentication related interfaces.
  *
  */
-public class DefaultDeviceResolver implements ApplicationLevelInfoSupplier, AdvancedPskStore, AdvancedApplicationLevelInfoSupplier {
-
-    /**
-     * Key in the extended info of the peer identity containing the auth-id used for authentication.
-     */
-    public static final String EXT_INFO_KEY_HONO_AUTH_ID = "hono-auth-id";
-    /**
-     * Key in the extended info of the peer identity containing the authenticated Device.
-     */
-    public static final String EXT_INFO_KEY_HONO_DEVICE = "hono-device";
+public class DefaultDeviceResolver implements AdvancedPskStore, AdvancedApplicationLevelInfoSupplier {
```

Comment:
shouldn't we rename this to e.g. `PskDeviceResolver` now that we also have a `CertificateDeviceResolver`?

Reply:
Done.